From 637bd699c08ac7d0b9b022a021bc1a318fbed530 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 1 Aug 2023 17:41:21 -0700
Subject: [PATCH 1/3] Update sudoers default config

Add wheelnopw group and read the adminconfdir drop-in files.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 plugins/sudoers/sudoers.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/plugins/sudoers/sudoers.in b/plugins/sudoers/sudoers.in
index 914d45e..bea3840 100644
--- a/plugins/sudoers/sudoers.in
+++ b/plugins/sudoers/sudoers.in
@@ -88,6 +88,9 @@ root ALL=(ALL:ALL) ALL
 ## Uncomment to allow members of group wheel to execute any command
 # %wheel ALL=(ALL:ALL) ALL
 
+%wheel ALL=(ALL) ALL
+%wheelnopw ALL=(ALL) NOPASSWD:ALL
+
 ## Same thing without a password
 # %wheel ALL=(ALL:ALL) NOPASSWD: ALL
 
@@ -101,3 +104,5 @@ root ALL=(ALL:ALL) ALL
 
 ## Read drop-in files from @sysconfdir@/sudoers.d
 @includedir @sysconfdir@/sudoers.d
+
+@includedir @adminconfdir@/sudoers.d
-- 
2.41.0

